README.md

```diff
@@ -55,6 +55,16 @@ yay -S python-launcher
 
 https://aur.archlinux.org/packages/python-launcher
 
+#### [Fedora Linux](https://getfedora.org/)
```

Comment:
This is already in the Linux section, so no need to specify that Fedora is a Linux distro.

```suggestion
#### [Fedora](https://getfedora.org/)
```

Reply:
Fedora Linux is the official name of the distro that Fedora produces. There has been some discussion around this in the Fedora community and the consensus is to call the distro Fedora Linux where possible. Relevant change proposal: https://fedoraproject.org/wiki/Changes/Fedora_Linux_in_os-release